### ISSUE TITLE
Exceptions not displayed correctly

### ISSUE THREAD

**Issue by ciaranmcnulty** [phpspec/phpspec#993]
When presenting an exception type it's wrapped in <label> tags that aren't then parsed into colours, making them harder to read.

This also happens in non-decorated output which is definitely incorrect.

**Comment by ciaranmcnulty** [comment]
Example:
![screen shot 2016-09-01 at 12 46 43](https://cloud.githubusercontent.com/assets/237866/18166288/31e017d2-7042-11e6-8f01-0ea257587c9e.png)

**Comment by ciaranmcnulty** [comment]
The colouring is addressed by #994 but there is still the issue of the tagged version being used for the undecorated output:

```
PhpSpec/Foo                                                                                    
  11  - it is initializable
      exception [exc:Exception("exception")] has been thrown.

/  skipped: 0%  /  pending: 0%  /  passed: 99%  /  failed: 0%   /  broken: 1%   /  771 examples
131 specs
771 examples (770 passed, 1 broken)
5218ms
```
